output.go

```diff
@@ -24,6 +25,69 @@ Txout-script / scriptPubKey   Script                                      <out-s
 type Output struct {
 	Satoshis      uint64
 	LockingScript *bscript.Script
+	index         int
+}
+
+type outputJSON struct {
+	Value        float64 `json:"value"`
+	Satoshis     uint64  `json:"satoshis"`
+	Index        int     `json:"n"`
+	ScriptPubKey struct {
```

Comment:
On the Output?

Reply:
yes please, moving forward we should always use `lockingScript` instead of `scriptPubKey` and `unlockingScript` instead of `scriptSig`, but still support the legacy terms until we can deprecate and phase them out in the future...